sig/temporal/worker.rbs

```diff
@@ -18,9 +19,16 @@ module Temporal
 
     attr_reader mutex: Thread::Mutex
     attr_reader runtime: Temporal::Runtime
-    attr_reader activity_worker: Temporal::Worker::Activity
+    attr_reader activity_worker: Temporal::Worker::ActivityWorker?
```

Comment:
Is this public?

Reply:
Nope, it's private (as indicated by line 18)